Resources/Prototypes/Chemistry/mixing_types.yml

```diff
@@ -1,3 +1,36 @@
+# Default Mixing
+# Not actually used in reactions: only meant for guidebook display purposes.
+
+- type: mixingCategory
+  id: DefaultMix
+  verbText: mixing-verb-default-mix
+  icon:
+    sprite: Objects/Specific/Chemistry/beaker_large.rsi
+    state: beakerlarge
+
+- type: mixingCategory
+  id: DefaultGrind
+  verbText: mixing-verb-default-grind
+  icon:
+    sprite: Structures/Machines/juicer.rsi
+    state: juicer0
+
+- type: mixingCategory
+  id: DefaultJuice
+  verbText: mixing-verb-default-juice
+  icon:
+    sprite: Structures/Machines/juicer.rsi
+    state: juicer0
+
+- type: mixingCategory
+  id: DefaultCondense
+  verbText: mixing-verb-default-condense
+  icon:
+    sprite: Structures/Piping/Atmospherics/condenser.rsi
+    state: display
```

Comment:
this is really sus

Reply:
It basically just allows a bunch of the logic to be reused rather than special-cased for different reaction types.

Also helps in that it defines data in YAML rather than just shoving it in a CS file somewhere.

Reply:
i think probably call them `Dummy*` instead of `Default*` and make it more obvious you shouldnt use these